lib/Newsletter/Sending/SendingQueuesRepository.php

```diff
@@ -45,4 +46,36 @@ public function isSubscriberProcessed(SendingQueueEntity $queue, SubscriberEntit
     if (is_null($task)) return false;
     return $this->scheduledTaskSubscribersRepository->isSubscriberProcessed($task, $subscriber);
   }
+
+  /**
+   * @return SendingQueueEntity[]
+   */
+  public function findAllForSubscriberSentBetween(
+    SubscriberEntity $subscriber,
+    ?\DateTimeInterface $dateTo,
+    ?\DateTimeInterface $dateFrom
+  ): array {
+    $qb = $this->entityManager->createQueryBuilder()
+      ->select('s, n')
+      ->from(SendingQueueEntity::class, 's')
+      ->join('s.task', 't')
+      ->join('t.subscribers', 'tsub')
+      ->join('tsub.subscriber', 'sub')
```

Comment:
I think that you don't need this join. You can check the subscriber by the previous join with the alias `tsub`.
Something like this `->andWhere(`tsub.subscriber = :subscriber')`